web/views/layout.erb

```diff
@@ -29,5 +28,10 @@
     </div>
     <%= erb :_footer %>
     <%= erb :_poll_js %>
+    <script type="text/javascript">
```

Comment:
We pass a variable from Rails to the JavaScript app to load the correct locale (`<%= locale %>`).
I don't see how we can have the JS app use the same locale as Rails without passing this through.